kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/network/TabWirelessUi.java

```diff
@@ -315,6 +315,12 @@
     HelpButton channelListHelp;
     @UiField
     HelpButton countryCodeHelp;
+    @UiField
+    Modal regDomModal;
```

Comment:
I would name this regDomErrorModal to make it clear that it pops out only on error